BRB/PushButton.py

```diff
@@ -232,26 +204,26 @@ def RNA(config, group, project, organism, libraryType, tuples):
     Need to set --libraryType
     """
     project = BRB.misc.pacifier(project)
-    outputDir = createPath(config, group, project, organism, libraryType, tuples)
+    org_name, org_label, org_yaml = organism
+    outputDir = createPath(config, group, project, org_label, libraryType, tuples)
     if os.path.exists(os.path.join(outputDir, "analysis.done")):
         return outputDir, 0, False
     PE = linkFiles(config, group, project, outputDir, tuples)
-    org = organism2Org(config, organism)
     CMD = "PATH={}/bin:$PATH".format(os.path.join(config.get('Options', 'snakemakeWorkflowBaseDir')))
-    CMD = [CMD, 'mRNAseq', '--DAG', '--trim', '-i', outputDir, '-o', outputDir, org]
+    CMD = [CMD, 'mRNAseq', '--DAG', '--trim', '-i', outputDir, '-o', outputDir, org_yaml]
     if tuples[0][2].startswith("Smart-Seq2"):
         # SMART-seq isn't a dUTP-based method!
         CMD.extend(['--libraryType', '0'])
     elif tuples[0][2].startswith("NEBNext Low Input RNA Library"):
         # Unstranded
-        CMD.extend(['--libraryType', '0'])
+        CMD.extend(['--libraryType', '0', r"--trimmerOptions '-a AGATCGGAAGAGC -A AGATCGGAAGAGC'"])
```

Comment:
This comment is only to highlight that I sneaked in a change that should probably have had its own PR.